libs/vm/src/module.cpp

```diff
@@ -416,6 +416,11 @@ Module::Module()
       [](VM *vm, TypeId) -> Ptr<Fixed128> {
         return Ptr<Fixed128>{new Fixed128(vm, fixed_point::fp128_t::_0)};
       });
+
+  // GetClassInterface<UInt256Wrapper>().CreateSerializeDefaultConstructor(
```

Comment:
TODO(issue number) if you're not going to remove these commented out lines

Reply:
added